server.go

```diff
@@ -67,6 +68,7 @@ type Server struct {
 
 	cfg             *Config
 	db              *clientdb.DB
+	fundingManager  *funding.Manager
```

Comment:
Was there a reason for it (and the other managers) living in the RPC server instead of here similar to how it's done in lnd?

Reply:
I'm not sure. I think it just grew that way organically because only the RPC server needed access to them.